windows/generate_compiled_windows_agent.sh

```diff
@@ -57,6 +63,14 @@ main() {
         "-h"|"--help")
             help 0
             ;;
+        "-cv"|"--cv2pdb-path")
+            if [ -n "$2" ]; then
+                DEBUG_SYMBOLS_PATH="$2"
```

Comment:
`DEBUG_SYMBOLS_PATH` variable name not seems to be a tools path. Output path should still be the same of the binary package
Could you please change it to `CV2PDB_TOOLS_PATH` or something similar?